data/map.txt

```diff
@@ -25390,7 +25390,7 @@ planet "Into White"
 planet "Karek Fornati"
 	attributes korath efret
 	landscape land/desert1
-	description `Karek Fornati was once a farming world, before global temperatures began to rise and the overworked farmland slowly deteriorated into deserts. The Korath who remain have dedicated their lives to finding a way to keep the "sand seas" from swallowing up the last of the fertile land, but without much success, and there are regions near the equator where sandstorms rage continually and no living things can survive.`
+	description `Karek Fornati was once a farming world, before global temperatures began to rise and the overworked farmland slowly deteriorated into deserts. The Korath who remain have dedicated their lives to finding a way to keep the "sand seas" from swallowing up the last of the fertile land, but without much success, and there are regions near the equator where sandstorms rage continually and neither animal nor plant can survive.`
```

Comment:
This just seems unnecessarily verbose, now. Reading this, I would simply think "why not shorten that as 'no living thing(s)'?" This description is probably just better off being left alone.

Reply:
Done. If MZ likes the "neither animals nor plants" language we can always put it back.